docs/book/v3/cookbook/access-common-data-in-templates.md

```diff
@@ -0,0 +1,75 @@
+# How Can I Access Common Data In Templates?
+
+This is a question that's asked a lot of times. How can I make common data like
```

Comment:
We can omit the first sentence; the fact that we have a recipe is indicative that it's a frequently asked question.